cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -109,19 +108,52 @@ Downloading and Distributing Packages
 
 Downloading and Installing CSD
 ------------------------------
-- Download the CDAP CSD (Custom Service Descriptor) by `downloading the JAR file 
-  <http://cask.co/resources/#cdap-integrations>`__.
+To install CDAP on a cluster managed by Cloudera, we have available a Custom Service
+Descriptor (CSD) which you can install onto your CM server. This adds CDAP to the list of
+available services which CM can install. 
+
+.. _cloudera-compatibility-matrix:
+
++----------------------------------------------------------------------------------------------------------------------+
+| Supported Cloudera Manager (CM) and Cloudera Data Hub (CDH) Distributions                                            |
++------------+---------------------+---------------------+----------------------------+--------------------------------+
+| CM Version | CDH Versions        | CSD Versions        |  CDAP Parcel               | CDAP HBase Coprocessor Version |
++============+=====================+=====================+============================+================================+
+| 5.5        | no greater than 5.5 | 3.0.x through 3.3.x | *Matching CSD major.minor* | ``hbase10cdh550``              |
```

Comment:
Every column in this row is accurate by itself, however taken together as an entire row it is a little misleading, since CDAP < 3.3 will not work against CDH 5.5.

The dependencies are as follows (seems almost circular):
CDH versions can only be installed on certain CM versions
CSDs can only be installed on certain CM versions (either explicit or implicitly via feature usage)
CDAP parcel major.minor should match CSD
CDAP versions must be compatable with CDH version